subsys/bluetooth/host/l2cap.c

```diff
@@ -701,6 +701,11 @@ static void le_conn_param_update_req(struct bt_l2cap *l2cap, uint8_t ident,
 		return;
 	}
 
+	if (conn->state == BT_CONN_DISCONNECTING) {
+		BT_WARN("Device is disconnecting");
```

Comment:
can you rather do `!= BT_CONN_STATE_CONNECTED` and say `Not connected` in the message?

Reply:
Should be resolved in https://github.com/zephyrproject-rtos/zephyr/pull/48942/commits/acc95d5c4b49f23b2bddeddd75c62ba700d69e73